photon-ml/src/test/scala/com/linkedin/photon/ml/util/IOUtilsTest.scala

```diff
@@ -14,33 +14,63 @@
  */
 package com.linkedin.photon.ml.util
 
+import java.io.{File, FileOutputStream}
+
 import com.linkedin.photon.ml.test.TestTemplateWithTmpDir
 import org.apache.hadoop.conf.Configuration
 import org.testng.Assert
 import org.testng.annotations.{DataProvider, Test}
-
 import scala.collection.mutable.ArrayBuffer
-
+import scala.io.Source
```

Comment:
Fixed in files for this checkin. I like that order. java - javax - scala - external libs - linkedin, from general to particular.